uint16 zmp_P0
uint16 zmp_P1
uint16 zmp_P2
uint16 zmp_P3
uint16 zmp_P4
uint16 zmp_P5
uint16 zmp_P6
uint16 zmp_P7
float32 body_roll
float32 body_pitch
float32 body_yaw

